client/gutenberg/extensions/presets/jetpack/index.json

```diff
@@ -4,7 +4,8 @@
     "map",
     "markdown",
     "publicize",
-    "simple-payments"
+    "simple-payments",
+    "subscriptions"
```

Comment:
Since this block isn't yet ready for production and is still in progress, should we add it to the `beta` list instead?